deploy/helm/self-service-agent/values.yaml

```diff
@@ -90,8 +90,7 @@ llama_stack_url: http://llamastack:8321
 sessionSecret: {}
  # value: "my-session-secret"
 
-seed:
-  admin_user:
-    username: kubeadmin
+#seed:
```

Comment:
Should we just delete instead of comment out? I guess same applies to all commented out stuff?